source/process/release-process.md

```diff
@@ -268,23 +284,33 @@ If a bug fix release is required, run through the following steps:
 
 Once final dot release build is ready to cut:
 
-1. Build:  
+1. Dev:
+    - Verify hashes and GPG signatures are correct, once build is cut
+    - Delete RCs after final version is shipped
+    - Merge changes made to release branch into vXXX
```

Comment:
Propose `Merge changes made to release branch into master`

Reply:
Done